src/main/java/com/dreammaster/gthandler/recipes/LaserEngraverRecipes.java

```diff
@@ -255,28 +255,28 @@ public void run() {
         // Optical to Wafer Buff Recipes
         GT_Values.RA.addLaserEngraverRecipe(
                 ItemList.Circuit_Silicon_Wafer6.get(1L),
-                GT_OreDictUnificator.get(OrePrefixes.lens, Materials.NetherStar, 1L).copy(),
+                GT_OreDictUnificator.get(OrePrefixes.lens, Materials.NetherStar, 1L).copy().splitStack(0),
```

Comment:
why not set 1L to 0L ?
like in this recipe https://github.com/GTNewHorizons/NewHorizonsCoreMod/pull/600/files#diff-99457ece25a9877f6a71a1e17dffcebe33cc56c5d36d69b707b515bfbce3edcfL219

Reply:
i don't know, @TheEpicGamer274  wrote the code

Reply:
Use 0L as dream suggested.

Reply:
GT_OreDictUnificator would just give null

Reply:
you could use GT_Utility.copyAmount(0, GT_OreDictUnificator.get(OrePrefixes.lens, Materials.NetherStar, 1)) though to keep it consistent with the rest of the file

Reply:
Hmm, we should change that possibly or what do you think about it?

Reply:
I agree with chochem

Reply:
ye i'm testing it with chochems idea

Reply:
it would just crash if i gave it 0L

Reply:
sounds about right

Reply:
I believe my water line recipes have the exact thing for lens use that is the best to use usually